log4j-over-slf4j/pom.xml

```diff
@@ -41,4 +41,21 @@
     </dependency>
   </dependencies>
 
+  <build>
+    <plugins>
+
+      <plugin>
+        <groupId>org.apache.felix</groupId>
+        <artifactId>maven-bundle-plugin</artifactId>
+        <configuration>
+          <instructions>
+            <Import-Package>org.slf4j.*;version="$${range;[===,3);${slf4j.api.minimum.compatible.version}}"</Import-Package>
```

Comment:
Is it really necessary for this artifact to specify a lower bound for the slf4j package.imports of 1.6?